src/docs/Foundations/Layout/Grid/index.mdx

```diff
@@ -7,11 +7,11 @@ status:
   scss: 'stable'
 ---
 
-> A grid is used to create a **more structured and more harmonious design.** It’s the frame of your page, you build the layout by placing blocks of contents (text, images,…) on a certain number of columns. Gutters are used to separate 2 blocks, they give air to the design. Columns and gutters form a container. The container can be **full width or with a fixed size.**
+> A grid is used to create a structured and harmonious design for every breakpoints. It is the frame of your page that allows you to build your layout by placing blocks of contents (texts, images, etc.) using a certain number of columns. Gutter are used to give some blank space between 2 separate blocks. A container is the combination of columns and gutters. The container can be full width (fluid) or have a fixed size (fixed container).
```

Comment:
`(texts, images, etc.)` → Remove the dot 🙂

Reply:
From my point of view, this part seems useless: `Gutter are used to give some blank space between 2 separate blocks. A container is the combination of columns and gutters. The container can be full width (fluid) or have a fixed size (fixed container).`.
The grid anatomy explains the same things.

Reply:
etc. is an abbreviation that needs the dot.